### ISSUE TITLE
Switch to FastParse

### ISSUE THREAD

**Issue by tabdulradi** [cassandra-scala/troy#121]
http://www.lihaoyi.com/fastparse/#GettingStarted

**Comment by tabdulradi** [comment]
Before we can pick this task, we need to ensure 100% coverage under cql-ast and cql-parses sub-projects.

**Comment by tabdulradi** [comment]
This task is on hold now. Please don't pick it up.
